### PR TITLE
runtime theme swapping

### PR DESCRIPTION
**Pull request by brianleach**
Jul-29-2021 11-34-32

### CODE REVIEW

File: src/theme/colors/ER.ts [hunk 1]
```diff
@@ -2,14 +2,14 @@ import type { Colors } from './colors.types';
...
   primary: {
-    100: '#D1EBCE',
-    200: '#A3D69E',
-    300: '#75C26D',
-    400: '#4B9743',
-    500: '#367A30',
-    600: '#206B19',
-    700: '#1A5614',
-    800: '#13400F',
+    100: '#FF0000',
```
Comment (brianleach) [inline src/theme/colors/ER.ts:5]: making everything red just to make it obvious

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import { Global, ThemeProvider } from '@emotion/react';
 import RecipeCard from './components/molecules/RecipeCard';
-import theme from './theme';
+import getTheme from './theme';
 import { generateFontFaces } from './theme/helpers/generateFontFaces';
 
 function App() {
+  // const theme = getTheme('ER');
+  const theme = getTheme(); // put theme in state?
```
Comment (inkblotty) [inline src/App.tsx:8]: Or memoize it: `useMemo(() => getTheme(), [])`

Comment (inkblotty) [reply src/App.tsx:8]: I guess if you could swap out the brand code it should be state

**Review by inkblotty — COMMENTED** [review]
The runtime branded-ness is good, but if we're gonna use backups, can we make backup files? Even if they return the same colors, etc. -- that way the naming conventions are clearer and not a source of confusion.

**Review by inkblotty — APPROVED** [review]

**Pull request merged**